app/src/main/java/camp/nextstep/edu/calculator/MainPresenterImpl.kt

```diff
@@ -0,0 +1,49 @@
+package camp.nextstep.edu.calculator
+
+import camp.nextstep.edu.domain.CAST_INT_EXCEPTION
+import camp.nextstep.edu.domain.Calculator
+import camp.nextstep.edu.domain.Expression
+import camp.nextstep.edu.domain.Operator
+
+class MainPresenterImpl(
```

Comment:
```suggestion
class MainPresenter(
```
Impl 라는 접미사는 사용하지 않아도 좋을거같아요!